Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,28 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            string firstName = "Tom";
+            string lastName = "Jefferson";
+            Decimal prisonerAge = 19;
+            Decimal weight = 50;
+            Decimal height = 156.5m;
+            Decimal bodyMassIndex = (weight / height / height) * 10000;
+
+            Console.WriteLine(firstName + " " + lastName + ", is " + prisonerAge + " years old, " + weight + " kg, and " + height + " cm.");
+            Console.WriteLine("Prisoner BMI: " + bodyMassIndex);
+            Console.ReadLine();
+
+            string firstNameTu = "Kai";
+            string lastNameTu = "the Nigerian C# prince";
+            Decimal prisonerAgeTu = 25;
+            Decimal weightTu = 69;
```

Comment:
You should always use the most appropriate data type for your data. Using decimal for the age is not neccessary at all.
Also decimal is more commonly used for money and is not neccessary for data like height or weight. You can easily use float here.

Reply:
FIxed.

> Important: for money or scientific calculations use decimal.

I was under the impression that calculation the BMI was a scientific thing. That is why I used Decimal.

Reply:
Oh yeah but I guess that scientific means like you need it very very very exact (without calculation errors) rather than having a scientific context for to it.